modules/javafx.base/src/main/java/com/sun/javafx/PlatformUtil.java

```diff
@@ -71,6 +71,7 @@
     private static final boolean LINUX = os.startsWith("Linux") && !ANDROID;
     private static final boolean SOLARIS = os.startsWith("SunOS");
     private static final boolean IOS = os.startsWith("iOS");
+    private static final boolean STATIC_BUILD = "Substrate VM".equals(System.getProperty("java.vm.name"));
```

Comment:
That's a good question. At this moment, there is a 1-1 link between statically linked images and Substrate VM. 
In case there will be more VM's that are statically linked with the applications and resources, I assume it will be beneficial to add a property in the "java." space to reflect this. There are more parts in OpenJDK that can benefit from different treatment in case of a statically linked image.